tests/config/functional/setup_cfg/issue_3630/2.2.out

```diff
@@ -0,0 +1,2 @@
+************* Module Command line or configuration file
```

Comment:
What is this testing? This seems a little redundant with all the other tests that check for (incorrect) section headers.

Reply:
Just took all the example from the issue :)